src/AuthService.Token/TokenSettings.cs

```diff
@@ -5,7 +5,8 @@
     /// </summary>
     public class TokenSettings
     {
-        public double TokenLifetimeInMinutes { get; set; }
+        public double AccessTokenLifetimeInMinutes { get; set; }
```

Comment:
fix name -- AccessTokenLife**Time**InMinutes

Reply:
That name already was there in settings. I thought it is not so bad, because lifetime is one word